reference-architectures/deploy-arch-ibm-devsecops-alm.md

```diff
@@ -28,30 +28,22 @@ use-case: DevOps
 
 industry: Technology
 
-compliance:
-
 content-type: reference-architecture
 
 ---
 
-<!--
-The following line inserts all the attribute definitions. Don't delete.
--->
 {{site.data.keyword.attribute-definition-list}}
 
-# Title
+# DevSecOps application lifecycle management
 {: #deploy-arch-ibm-devsecops-alm}
 {: toc-content-type="reference-architecture"}
 {: toc-industry="Technology"}
 {: toc-use-case="DevOps"}
-{: toc-compliance=""}
 
-The DevSecOps deployable architecture creates a set of DevOps Toolchains and pipelines. DevSecOps uses Continuous Delivery (Git Repos and Issue Tracking, Tekton Pipelines,
-DevOps Insights, and Code Risk Analyzer), Secrets Manager, Key Protect, Cloud Object Storage, Container Registry and Vulnerability Advisor. Out of the box, DevSecOps also
-leverages popular scanning tools such as SonarQube, GoSec, OWASP Zap (dynamic scan), any unit test framework, and GPG signing. It can also be used with more tools such as
-external Git providers and artifact stores. DevSecOps supports hybrid deployments, in particular by using private pipeline workers, and can be interfaced with other
-deployment tools such as Satellite Config.
+The DevSecOps deployable architecture creates a set of DevOps Toolchains and pipelines. DevSecOps uses Continuous Delivery (Git Repos and Issue Tracking, Tekton Pipelines, DevOps Insights, and Code Risk Analyzer), Secrets Manager, Key Protect, Cloud Object Storage, Container Registry and Vulnerability Advisor.
+{: shortdesc}
```

Comment:
What's the {:shortdesc} here for .... a title for the next paragraph ?

Reply:
@gmarjoram - It's markup that's required for IBM Cloud docs. It identifies the preceding paragraph as the content that's included in SEO. Everything that Jenifer changed in this topic was to make the content follow the markup and language guidelines. I hope this helps!

Reply:
Thanks ... it does 👍